zmoney/Models/Constants/Notifications.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Notifications.swift
+//  zmoney
+//
+//  Created by Pavel Romanishkin on 20.04.21.
+//
+
+import Foundation
+
+public enum ZMoneyNotifications {
+  public static let tokenUpdated = "ZMoney.tokenUpdated"
+}
+
+extension Notification.Name {
+  public static let zMoneyConfigUpdated = Notification.Name(rawValue: ZMoneyNotifications.tokenUpdated)
```

Comment:
```suggestion
  public static let zMoneyConfigUpdated = Notification.Name(rawValue: "ZMoney.tokenUpdated")
```

Можно еще упростить: удалить `ZMoneyNotifications` и везде использовать `. zMoneyConfigUpdated`